apps/core/src/components/stake/StakedCard.tsx

```diff
@@ -49,17 +51,50 @@ export function StakedCard({
         );
     }, [validatorAddress, data]);
 
+    const { data: inactiveValidatorData } = useQuery({
+        queryKey: [data?.inactivePoolsId, validatorAddress],
+        async queryFn() {
+            if (!data?.inactivePoolsId || !validatorAddress) {
+                throw Error('Missing params');
+            }
+            const inactiveValidators = await iotaClient.getDynamicFields({
+                parentId: normalizeIotaAddress(data?.inactivePoolsId),
+            });
+
+            const pendingInactiveValidatorsData = await Promise.all(
+                inactiveValidators.data.map(
+                    async (validator) =>
+                        await getInactiveValidatorsData(iotaClient, validator.objectId),
+                ),
+            );
+
+            return pendingInactiveValidatorsData;
+        },
+        enabled: !!data?.inactivePoolsId && !!validatorAddress,
+        select(validators) {
+            return validators.find((validator) => validator?.validatorAddress === validatorAddress);
+        },
+    });
+
+    const combinedValidatorData = useMemo(() => {
+        if (!validatorMeta && !inactiveValidatorData) return null;
+        return {
+            ...validatorMeta,
+            ...inactiveValidatorData,
+        };
+    }, [validatorMeta, inactiveValidatorData]);
```

Comment:
Don't you maybe want to do something like this?

(No need for `useMemo` this is a cheap operation btw)

```suggestion
    const validatorData = validatorMeta || inactiveValidatorData || null;
```